microk8s-resources/wrappers/run-containerd-with-args

```diff
@@ -2,6 +2,11 @@
 
 set -ex
 
+# Re-exec outside of apparmor confinement
+if [ -d /sys/kernel/security/apparmor ] && [ "$(cat /proc/self/attr/current)" != "unconfined" ]; then
+    exec aa-exec -p unconfined -- "$0" "$@"
+fi
```

Comment:
Note, this is not a proper solution for strict mode.

Reply:
Yes, for strict confinement we need containerd confined.